docs/elements/reference/sign-up.mdx

```diff
@@ -0,0 +1,227 @@
+---
+description: Reference documentation for Clerk Elements sign-up components.
+---
+
+# Sign-up components
+
+The following components are used when creating sign-up flows. They are imported from `@clerk/elements/sign-up`. It is recommended to import them all under the `SignUp.` namespace to make discovery easier and reduce naming conflicts with other components throughout your application. The code snippets on this page assume you have imported the components this way.
```

Comment:
Okay I think you can apply my suggestions from `/docs/elements/reference/sign-in.mdx` above to this page 👍